src/AntiPlagiarism/AntiPlagiarism.Vsix/source.extension.vsixmanifest

```diff
@@ -1,26 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="Acumatica.AntiPlagiarism.7ae0340a-a17e-42a1-9335-0f26f5527693" Version="1.0" Language="en-US" Publisher="Acumatica" />
+        <Identity Id="Acumatica.AntiPlagiarism.7ae0340a-a17e-42a1-9335-0f26f5527693" Version="1.3" Language="en-US" Publisher="Acumatica" />
         <DisplayName>AntiPlagiarism</DisplayName>
-        <Description xml:space="preserve">AntiPlagiarism is a  tool for Visual Studio that allows to analyze solutions for copypasted code from a provided reference solution. The tool was designed to use the  Acumatica Framework as a reference solution but can be used with other solutions unrelated to Acumatica. </Description>
```

Comment:
We have Antiplagiarism in release notes and AntiPlagiarism here. Which is correct?

Reply:
good point. I think it's AntiPlagiarism. I'll fix it

Reply:
Fixed